### PR TITLE
feat: batched Nx.eye

### PR DESCRIPTION
**Pull request by polvalente**
related to #174

### CODE REVIEW

File: torchx/mix.exs [hunk 1]
```diff
@@ -56,8 +56,8 @@ defmodule Torchx.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.4.0"},
-      # {:nx, path: "../nx"},
+      # {:nx, "~> 0.4.0"},
+      {:nx, path: "../nx"},
```
Comment (josevalim) [inline torchx/mix.exs:60]: We can still use `{:nx, "~> 0.4.0"}`, given the previous functionality is a superset. :)

File: exla/mix.exs [hunk 2]
```diff
@@ -52,8 +52,8 @@ defmodule EXLA.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.4.0"},
-      # {:nx, path: "../nx"},
+      # {:nx, "~> 0.4.0"},
+      {:nx, path: "../nx"},
```
Comment (josevalim) [inline exla/mix.exs:56]: We can still use `{:nx, "~> 0.4.0"}`, given the new functionality is a superset. :)

**Review by josevalim — APPROVED** [review]

File: torchx/mix.exs (hunk 1 above)
Comment (polvalente) [reply torchx/mix.exs:60]: This PR also extends to accept rectangular identity shapes, so we can't test the updated implementations in EXLA and Torchx without pointing to latest Nx

Comment (polvalente) [reply torchx/mix.exs:60]: I'll defer the merge to you :)

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**